helion/runtime/kernel.py

```diff
@@ -105,6 +111,25 @@ def __init__(
             else:
                 self._annotations.append(ann)
 
+    def _get_bound_kernel_cache_key(
+        self, args: tuple[object, ...], signature: tuple[Hashable, ...]
+    ) -> BoundKernelInMemoryCacheKey | None:
+        extra_fns = self._specialize_extra.get(signature)
+        if extra_fns is not None:
+            extra_results: tuple[Hashable, ...] = tuple(s(args) for s in extra_fns)
```

Comment:
```suggestion
            extra_results: tuple[Hashable, ...] = tuple([s(args) for s in extra_fns])
```

Generators are slow:
```
>>> timeit.timeit("tuple(x for x in range(3))")
0.2792648430040572
>>> timeit.timeit("tuple([x for x in range(3)])")
0.1105302960058907
>>> 
```